### PR TITLE
Update Assert Statements For New TF 2.2.0 DLC

### PR DESCRIPTION
**Pull request by NihalHarish**
### Description of changes:
- Update asserts to work with the latest DLC

#### Style and formatting:

I have run `pre-commit install` to ensure that auto-formatting happens with every commit.

#### Issue number, if available

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tests/zero_code_change/test_tensorflow2_gradtape_integration.py [hunk 1]
```diff
@@ -26,10 +25,13 @@ def get_keras_data():
...
 
 
-def helper_test_keras_v2_gradienttape(script_mode: bool = False, json_file_contents="{}"):
+def helper_test_keras_v2_gradienttape(
+    script_mode: bool = False, json_file_contents="{}", default=False
+):
     """ Test the default ZCC behavior of saving losses and metrics in eager and non-eager modes."""
     smd.del_hook()
     tf.keras.backend.clear_session()
+    is_tf_2_2 = True if tf.__version__ == "2.2.0" else False
```
Comment (vandanavk) [inline tests/zero_code_change/test_tensorflow2_gradtape_integration.py:34]: can we use the test util?

Comment (NihalHarish) [reply tests/zero_code_change/test_tensorflow2_gradtape_integration.py]: We can. I have reverted it

**Comment by NihalHarish** [comment]
> relax the asserts to >= so that the tests dont fail on vanilla

Since we do not currently test on a vanilla TF 2x container, I'm keeping the strict asserts

File: tests/zero_code_change/test_tensorflow2_gradtape_integration.py (hunk 1 above)
Comment (NihalHarish) [reply tests/zero_code_change/test_tensorflow2_gradtape_integration.py]: I have reverted this.

**Pull request merged**